# Descriptive Analysis – Phase 2
This section explores pricing, room type distribution,
and neighborhood-level listing activity.


In [4]:
import pandas as pd

listings = pd.read_csv(
    r"D:\PROJECT\Airbnb Listings Analysis\dataset\processed\clean_listings.csv"
)


In [5]:
listings.shape


(2511, 16)

In [6]:
listings.head()


id                                               name  host_id  \
0  155305                 Cottage! BonPaul + Sharky's Hostel   746673   
1  197263                       Tranquil Room & Private Bath   961396   
2  209068                                    Terrace Cottage  1029919   
3  246315                          Asheville Dreamer's Cabin  1292070   
4  314540  Asheville Urban Farmhouse Entire Home 4.6 mi t...   381660   

  host_name  neighbourhood   latitude  longitude        room_type  price  \
0   BonPaul          28806  35.578640 -82.595780  Entire home/apt   95.0   
1   Timothy          28806  35.577350 -82.638040     Private room   44.0   
2     Kevin          28804  35.617641 -82.551819  Entire home/apt   90.0   
3     Annie          28805  35.596150 -82.506350     Private room   61.0   
4       Tom          28806  35.585610 -82.627310  Entire home/apt  200.0   

   minimum_nights  number_of_reviews last_review  reviews_per_month  \
0               1                454  2025-06-14               2.69   
1               2                 87  2024-09-08               0.56   
2              30                 67  2025-05-03               0.40   
3               7                 53  2019-10-30               0.32   
4               1                 35  2025-06-13               0.22   

   calculated_host_listings_count  availability_365  number_of_reviews_ltm  
0                               8               162                     16  
1                               2                73                      5  
2                               1               268                      2  
3                               3                62                      0  
4                               1               139                     11

In [7]:
listings["price"].agg(["mean", "median", "min", "max", "std"])


mean       167.124652
median     127.000000
min         20.000000
max       1000.000000
std        127.182104
Name: price, dtype: float64

### Overall Pricing Insights

- The average nightly price in Asheville is approximately $167
- The median price is significantly lower at $127, indicating a right-skewed price distribution
- Most listings are priced in the mid-range, with a small number of high-priced properties pulling the average upward
- The wide standard deviation suggests substantial price variability across listings


In [8]:
listings["room_type"].value_counts()


room_type
Entire home/apt    2264
Private room        227
Hotel room           12
Shared room           8
Name: count, dtype: int64